defaults.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
     '@apostrophecms/images-widgets': {},
     '@apostrophecms/files': {},
     '@apostrophecms/files-widgets': {},
-    '@apostrophecms/soft-redirects': {}
+    '@apostrophecms/soft-redirects': {},
+    '@apostrophecms/overlay': {}
```

Comment:
Is it better to name it "overlay" or "busy"?

Reply:
"busy". Name it for the function, not its current appearance.